src/js/components/AnalysisPanel/VegaChart.js

```diff
@@ -165,32 +197,49 @@ export default class VegaChart extends Component {
             <div className='vega-chart_click-area' onClick={() => this.setState({ showDownloadOptions: false })}></div>
           }
           {component === 'Report' ?
-          <div className={component === 'Report' ? 'vega-chart_download-container-report' : 'vega-chart_download-container'}>
-            {/* <h3 className="vega-chart-label">{results.data.attributes.name}</h3> */}
-            <h3 className="vega-chart-label">{reportLabel}</h3>
-            <div className='vega-chart-menu-container'>
-              <div className='vega-chart-menu' onClick={() => console.log('clicked')}>
-                <SVGIcon className="vega-chart-menu-icon" id={'icon-gear'} />
-                {/* <span className="vega-chart-menu-text">SETTINGS</span> */}
-              </div>
-              <div className="vega-chart-divider"></div>
-              <div className='vega-chart-menu' onClick={() => this.setState({showDownloadOptions: !showDownloadOptions})}>
-                <SVGIcon className="vega-chart-menu-icon" id={'icon-download-grey'} />
+            <div className={component === 'Report' ? 'vega-chart_download-container-report' : 'vega-chart_download-container'}>
+              {/* <h3 className="vega-chart-label">{results.data.attributes.name}</h3> */}
+              <h3 className="vega-chart-label">{reportLabel}</h3>
+              <div className='vega-chart-menu-container'>
+                {
+                  (
+                  analysisId === 'TC_LOSS' ||
```

Comment:
@KaylaKremer Let's do this automatically by running some kind of `filter` on all of our analysisModules looking if they have some `uiModules` in their array!

Reply:
Made a PR to clean this up! https://github.com/wri/gfw-mapbuilder/pull/472